lib/fine-types/app/Options.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE ApplicativeDo #-}
```

Comment:
As you know, I'm fairly conservative as far as language extensions are concerned, but for this particular use case, I think that's worth it. 🤔